firefox-ios/firefox-ios-tests/Tests/XCUITests/DragAndDropTests.swift

```diff
@@ -110,11 +110,15 @@ class DragAndDropTests: BaseTestCase {
             checkTabsOrder(dragAndDropTab: true, firstTab: secondWebsite.tabName, secondTab: firstWebsite.tabName)
             if !iPad() {
                 let url = app.textFields[AccessibilityIdentifiers.Browser.UrlBar.url]
-                XCTAssert(
-                    secondWebsite.url.contains(url.value! as! String), "The tab has not been dropped correctly"
-                ) } else {
-                    XCTAssertEqual(app.otherElements["Tabs Tray"].cells.element(boundBy: 0).label, secondWebsite.tabName)
+
+                if let urlString = url.value as? String {
+                    XCTAssert(secondWebsite.url.contains(urlString), "The tab has not been dropped correctly")
+                } else {
+                    XCTFail("Failed to retrieve a valid URL from the UrlBar")
```

Comment:
Using `URL` could be misleading, because we have it as a string value, so to be exact, I'd write `Failed to retrieve a valid URL string from the UrlBar`. Please adjust it for the other case too.